api/types/constants.go

```diff
@@ -21,6 +21,17 @@ const (
 	// lets us to add different permission types
 	DefaultAPIGroup = "gravitational.io/teleport"
 
+	// DefaultReleaseServerAddr is the default release service URL
+	DefaultReleaseServerAddr = "rlz.teleport.sh"
+
+	// ReleaseServerEnvVar is the environment variable used to overwrite
+	// the default release server address
+	ReleaseServerEnvVar = "RELEASE_SERVER_HOSTPORT"
```

Comment:
nit: I don't have strong opinion but maybe: 
```suggestion
	ReleaseServerEnvVar = "RELEASE_SERVER_ADDRESS"
```